src/components/Hero/Hero.jsx

```diff
@@ -58,19 +78,38 @@ function Hero() {
 						<strong> {t("hero.at")} uOttawa</strong>
 					</h3>
 					<h3>
-						Mar &nbsp;
-						<time dateTime="03/02/2023">3</time>&ndash;
-						<time dateTime="05/02/2023">5</time>, <strong>2023</strong>
+						Feb &nbsp;
+						<time dateTime="02/02/2024">2</time>&ndash;
+						<time dateTime="04/02/2024">4</time>, <strong>2024</strong>
 					</h3>
 				</div>
 				<h1 id="Hero">
 					<img className={styles["banner-logo"]} src={BannerLogo} alt="Hack the Hill"></img>
 				</h1>
 				<h2>{t("hero.h2")}</h2>
-				<h3>{t("hero.h3")}</h3>
-				<Button href={t("hero.link")} target={"_blank"} offset={-160}>
-					{t("hero.more")}
-				</Button>
+				<form
+					className={styles["hero-form"]}
+					action={"https://tracker.hackthehill.com/follow?email=" + { email }}
+				>
+					<input
+						className={styles["hero-input"]}
+						onChange={handleEmailChange}
+						type="text"
```

Comment:
Why not `type="email"`? It will validate the email address without having to use custom regex